user-journey.adoc

```diff
@@ -0,0 +1,180 @@
++
+----
+mkdir istiooc && cd istiooc
+ 
+#osx
+wget -O oc https://github.com/openshift-istio/origin/releases/download/istio-3.9-0.7.1-alpha7/istiooc_darwin
+chmod +x oc 
+ 
+#linux
+wget -O oc https://github.com/openshift-istio/origin/releases/download/istio-3.9-0.7.1-alpha7/istiooc_linux
+chmod +x oc
+
+# Add the client to your path
+export PATH=$(pwd):$PATH #or add to PATH variable in profile
+----
+. Ensure that the latest origin-ansible docker image has been downloaded, just in case it was updated recently:
++
+----
+docker pull openshiftistio/origin-ansible:0.7.1
+----
+. User starts the cluster using --istio and --launcher-* as a parameters (These install Istio and the Red Hat Launcher into the OpenShift cluster, respectively). You will need a GitHub account and API Access Token. Click https://github.com/settings/tokens[here] to generate your github token.
++
+----
+oc cluster up \
+   --public-hostname=ETHERNET_IP_ADDRESS \
+   --istio \
+   --launcher \
+   --launcher-catalog-git-branch=GIT_REF_BOOSTERS_REPO \
+   --launcher-catalog-git-repo=GIT_HUB_BOOSTERS_REPO \
+   --launcher-openshift-user=OPENSHIFT_ADMIN_USER \
+   --launcher-openshift-password=OPENSHIFT_ADMIN_PWD \
+   --launcher-github-username=GITHUB_USER \
+   --launcher-github-token=GITHUB_TOKEN
+----
++
+You can also use the sensible defaults we have in place and provide only the following info:
++
+----
+oc cluster up \
+   --public-hostname=ETHERNET_IP_ADDRESS \
+   --istio \
+   --launcher \
+   --launcher-openshift-user=OPENSHIFT_ADMIN_USER \
+   --launcher-openshift-password=OPENSHIFT_ADMIN_PWD \
+   --launcher-github-username=GITHUB_USER \
+   --launcher-github-token=GITHUB_TOKEN
+----
+. User logs in to the recently created cluster
+oc login -u system:admin
+. User adds the cluster-admin role  to the admin user
++
+----
+oc adm policy add-cluster-role-to-user cluster-admin admin
+----
+. Log on using the admin user
++
+----
+oc login -u admin -p admin
+----
+. User waits untill the “launcher-backend” and “launcher-fronted” pods have been started
++
+----
+oc get pods -n devex -w
+NAME                          READY     STATUS    RESTARTS   AGE
+configmapcontroller-1-vh78r   1/1       Running   0          1m
+launcher-backend-2-vb4vt      1/1       Running   0          1m
+launcher-frontend-2-bc7n2     1/1       Running   0          1m	
+----
+. User logs into openshift console: 
+.. https://ETHERNET_IP_ADDRESS:8443 
+. User opens the launcher UI URL, which is one of the featured applications.
+.. http://launcher-devex.ETHERNET_IP_ADDRESS.nip.io:8843/
++
+IMPORTANT: Do NOT use HTTPS
+. Launcher UI opens
+. User clicks on the “launch your project” button, select the deployment type - Use OpenShift online, a runtime and select an istio mission (see video for the AB Testing Booster using Spring Boot - https://www.youtube.com/watch?v=hsd6ezE2tpg) 
+. User sets name of project (by convention it corresponds to the name of the openshift project and github repo), GAVs detailed
```

Comment:
GAVs?

Reply:
GAV = Maven Artififact -> GroupId - ArtifactId and Version

Reply:
Ah, I should know that. It's just been ages since I did anything with Maven.